adminSiteServer/IndexPage.tsx

```diff
@@ -11,12 +11,6 @@ export const IndexPage = (props: {
     isSuperuser: boolean
     gitCmsBranchName: string
 }) => {
-    const siteStylesheetUrls = [
-        "https://fonts.googleapis.com/css?family=Lato:300,400,400i,700,700i,900|Playfair+Display:400,600,700&display=swap",
-        webpackUrl("owid.css", "/admin"),
-        webpackUrl("commons.css", "/admin"),
-    ]
```

Comment:
@mlbrgl it seems this is unused. could you double-check that it is safe to remove?

Reply:
@marcelgerber ah yes, that's left over from one of the early attempts to isolate the preview. It can be deleted now.